RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRegistry.java

```diff
@@ -1726,6 +1731,7 @@ private BlackboardArtifact createAssociatedArtifact(String filePathName, Blackbo
 
         for(OsAccount account: tskCase.getOsAccountManager().getOsAccounts(((DataSource)dataSource).getHost())) {
             Optional<String> userName = account.getLoginName();
+            // @@@ BC: Seems like this should be calling account.getAddr() to get the SID. 
```

Comment:
This is not new code.  Changing this would effect existing code.

@markmckinnon and I felt like this was the correct thing to be doing.